build.sbt

```diff
@@ -74,15 +74,13 @@ dockerUpdateLatest := true
 defaultLinuxInstallLocation in Docker := s"/opt/${moduleName.value}"
 
 dockerCommands := Seq(
-  Cmd("FROM", "alpine:3.6"),
-  Cmd("RUN", "apk upgrade --update && apk add --update openjdk8-jre bash coreutils"),
+  Cmd("FROM", "advancedtelematic/alpine-jre:8u191-jre-alpine3.9"),
   ExecCmd("RUN", "mkdir", "-p", s"/var/log/${moduleName.value}"),
   Cmd("ADD", "opt /opt"),
   Cmd("WORKDIR", s"/opt/${moduleName.value}"),
   ExecCmd("ENTRYPOINT", s"/opt/${moduleName.value}/bin/${moduleName.value}"),
   Cmd("RUN", s"chown -R daemon:daemon /opt/${moduleName.value}"),
   Cmd("RUN", s"chown -R daemon:daemon /var/log/${moduleName.value}"),
-  Cmd("USER", "daemon")
```

Comment:
where is this coming from now in this case ?

Reply:
I had misread the docs. Fixed.